xml/admin_ceph_upgrade.xml

```diff
@@ -543,15 +543,31 @@ Nodes running older software versions must be upgraded in the following order:
 <screen>&prompt.cephuser;ceph config set osd bluestore_fsck_quick_fix_on_mount true</screen>
         <para>
           The OSD nodes will be converted automatically after their adoption
-          finishes.
+          finishes. The conversion may take minutes to hours, depending on how
+          much OMAP data the related hard disk contains. For more details, refer
+          to
+          <link xlink:href="https://docs.ceph.com/en/latest/releases/pacific/#upgrading-non-cephadm-clusters"/>.
         </para>
         <note>
           <para>
-            The conversion may take minutes to hours, depending on how much
-            OMAP data the related hard disk contains. For more details, refer
-            to
-            <link
-      xlink:href="https://docs.ceph.com/en/latest/releases/pacific/#upgrading-non-cephadm-clusters"/>.
+            You can also perform an offline compaction enabling a quick fix
+            and perform an online compaction on each OSD daemon that is
+            deployed on the OSD node identified by the
+            <replaceable>MINION_ID</replaceable>. For example
+          </para>
+<screen>
+&prompt.cephuser;cephadm unit --name osd.<replaceable>OSD_DAEMON_ID</replaceable> stop
+&prompt.cephuser;ceph-kvstore-tool bluestore-kv /var/lib/ceph/263302b4-afa2-11ec-a986-525400af093c/osd.<replaceable>OSD_DAEMON_ID</replaceable> compact
```

Comment:
This would be better as:

```suggestion
&prompt.cephuser;cephadm shell --name osd.<replaceable>OSD_DAEMON_ID</replaceable> ceph-kvstore-tool bluestore-kv /var/lib/ceph/osd/ceph-<replaceable>OSD_DAEMON_ID</replaceable> compact
```

(just because it means you don't have to _also_ replace that irritating FSID with whatever the actual cluster FSID is)